src/client/refresh/vk/vk_draw.c

```diff
@@ -169,6 +169,9 @@ refresh window.
 */
 void RE_Draw_TileClear (int x, int y, int w, int h, char *name)
 {
+	// This is handled in a special way in the Vulkan renderer. Clearing tiles
+	// are skipped and replaced by black bars during the UI render pass.
+#if 0
```

Comment:
@Yamagi Do we need clear function? Has this function used only for clean up borders?

Reply:
While analyzing a frame with renderdoc, I noticed these borders were being drawn in the wrong place at the beginning of the frame and, in any case, black borders were being drawn around the main world view later as part of the UI, so I think it's just not useful and commented it out apparently without visual consequences. :shrug: 

Maybe in the future the black borders from the UI can be replaced by this kind of tile?

Reply:
Maybe it related to my hack with [R_CleanupBorders]( https://github.com/yquake2/yquake2/blob/46c24d4/src/client/refresh/vk/vk_rmain.c#L1176). 

Could you please try to comment out this function and rerun? Maybe RE_DrawTileClear should do exactly such but it has regressed in some moment of time.

Reply:
It looks exactly such by debug prints in RE_DrawTileClear. :-(

When I have removed R_CleanupBorders in master code - borders is painted by correct placeholder. 

Could you please remove R_CleanupBorders at all?

Reply:
Sure, let me try to see what comes out and I'll report if there's any situation in which it's broken. We could handle those in a separate pull request.

Reply:
So from what I've seen, the border is broken with vk_pixel_size and, at least in the state of this branch, it's also being drawn in the wrong place with vid_fullscreen 1. It will need some more work before it can be fully restored.

Reply:
Fixed.